ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -581,6 +585,44 @@ def test_ipa_healthcheck_no_errors(self):
         )
         assert returncode == 0
 
+    def test_ipa_healthcheck_no_errors_with_overrides(self):
+        """
+        Test overriding command-line options in a configuration file.
+        """
+        tmpcmd = self.master.run_command(['mktemp'])
+        config_file = tmpcmd.stdout_text.strip()
+
+        self.master.put_file_contents(
+            config_file,
+            '\n'.join([
+                '[default]',
+                'output_type=human'
+            ])
+        )
+        returncode, output = run_healthcheck(
+            self.master, failures_only=True, config=config_file
```

Comment:
`run_healthcheck` is expecting json output (because the caller doesn't set output_type="human") but receives human output because of the config file. This triggers a JSONDecodeError in the call to `json.loads(result.stdout_text)`.

`run_healthcheck` needs to be adapted and take into account the content of the config file.

Reply:
This is testing a new feature in healthcheck where cli options can be set in configuration. But this release is only in rawhide currently which is why it failed.

Reply:
I meant that even with the new freeipa-healthcheck version it would fail. `run_healthcheck` has the following code:
```
        if output_type == "json":
            data = json.loads(result.stdout_text)
```
`output_type` is `json` by default (and not set here in line 602), and the code will call json.loads. But since the config file is used, ipa-healthcheck outputs human format and json.loads will complain that it doesn't recognize the output as valid JSON.